go_modules/spec/fixtures/projects/nonexisting_proxy/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/dependabot/vgotest
 
-go 1.18
+go 1.18 // TODO why was this already at 1.18?
```

Comment:
@jurre any idea? Was this accident or on-purpose? Just wondering if it's supposed to be always bumped to a non-existing version of go (which means the test needs changing because there's no way this will get remembered on every bump of go).

Reply:
I'm not sure 🤔 I see that I added this file, but I don't think it's relevant for the test. I think the main thing that we need to make sure of is that we don't change the  go version specified in the go.mod file. I don't think what is in there is used, we strip it out when copying over the file: https://github.com/dependabot/dependabot-core/blob/add0e471c1920e2a726273aee7f84a37429b31c6/go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb#L115-L127

Reply:
Ah, I see what this test is actually trying to test--that a non-existing proxy (`dependabot.com/nonexistingdep`) error gets handled correctly. This has nothing to do with the `go 1.18`. But should be safe enough to leave as-is (I'll remove the TODO note of course).